app/components/List.jsx

```diff
@@ -4,8 +4,19 @@ import React from 'react';
 import ListItem from './ListItem';
 import ReactCSSTransitionGroup from './TimeoutTransitionGroup';
 import { mergeAndPrefix } from '../utils/stylePropable';
+import { List } from 'material-ui';
+
+export default class extends React.Component {
+  static propTypes = {
```

Comment:
Do you mean like this?

```javascript
constructor(props) {
    super(props);

    this.propTypes = {
      itemData: React.PropTypes.array,
      onClick: React.PropTypes.func,
      style: React.PropTypes.object,
    };

    this.defaultProps = {
      style: {},
    };
  }
```

Console output: `150714/150556.048, [log,info], data: Server running at: http://localhost:8080
Warning: propTypes was defined as an instance property on _default. Use a static property to define propTypes instead.
Warning: React.createElement: type should not be null or undefined. It should be a string (for DOM elements) or a ReactClass (for composite components).`

Well I think we should stick with `static`. 

https://facebook.github.io/react/blog/2015/01/27/react-v0.13.0-beta-1.html#es7-property-initializers

Even if it's highly experimental nop? I mean, if it is working and the code is more readable. What could be better?